data/schemas/keyboard.jsonschema

```diff
@@ -150,6 +150,20 @@
                 "voices": {"type": "boolean"}
             }
         },
+        "auto_shift": {
+            "type": "object",
+            "properties": {
+                "enter": {"type": "boolean"},
```

Comment:
I kinda feel like all of the key configurations (eg, what it triggers and doesn't) should be in some subkey.  Not sure if that's the best way though

Reply:
I was of same thought. Initially I added all keys prefixed with `no_` under `exclusion` subkey. So it is obvious that those keys wouldn't trigger auto shift behavior. I also thought of using `behavior` as a key but it's pretty broad term

Reply:
I have updated PR with some changes and subkeys. Let me know if they make any sense. Or If there is a convention we'd be following.